pkgdb2/templates/package.html

```diff
@@ -394,13 +394,17 @@ <h4 title="`approveacls` ACL">Package Administrator(s)</h4>
               </form>
             </li>
           {% endif %}
-          {% if g.fas_user.username in admins or is_admin %}
+          {% if g.fas_user %}
             <li>
               <a href="{{ url_for('.update_acl',package=package.name,
                        update_acl='approveacls') }}"
-                title="Manage commit ACLs">
```

Comment:
So, this text was incorrect before, right?  The link url_for says 'approveacls', but this text says "commit".